models/repo_activity.go

```diff
@@ -262,35 +274,35 @@ func pullRequestsForActivityStatement(repoID int64, fromTime time.Time, merged b
 }
 
 // FillIssues returns issue information for activity page
-func (stats *ActivityStats) FillIssues(repoID int64, fromTime time.Time) error {
+func (stats *ActivityStats) FillIssues(repoID int64, fromTime time.Time, canReadPrivateIssues bool, userID int64) error {
 	var err error
 	var count int64
 
 	// Closed issues
-	sess := issuesForActivityStatement(repoID, fromTime, true, false)
+	sess := issuesForActivityStatement(repoID, fromTime, true, false, canReadPrivateIssues, userID)
```

Comment:
There are so many arguments for `issuesForActivityStatement`, should we introduce an struct to pass arguments?